functional-samples/cookbook.geolocation-popup/README.md

```diff
@@ -0,0 +1,20 @@
+This recipe shows how to get `geolocation` access within a popup
```

Comment:
Nit: Can we move this down between the header and "Overview"?

Reply:
fixed with [1dbac70](https://github.com/GoogleChrome/chrome-extensions-samples/pull/944/commits/1dbac7034801f93237ddfa43f3a9a44f9f7552eb)